packages/terra-status-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
```

Comment:
This should be under a Fixed section if the issue was logged as a bug

Reply:
Changed b469734